MaterialLib/Fluid/Viscosity/VogelsLiquidDynamicViscosity.h

```diff
@@ -0,0 +1,87 @@
+/*!
+   \file  VogelsLiquidDynamicViscosity.h
+   \brief Declaration of class for the pressure dependent viscosity model.
+
+   \author Wenqing Wang
+   \date Jan 2015
+
+   \copyright
+    Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+               Distributed under a Modified BSD License.
+               See accompanying file LICENSE.txt or
+               http://www.opengeosys.org/project/license
+ */
+#ifndef VOGELS_LIQUID_DYNAMIC_VISCOSITY_H_
+#define VOGELS_LIQUID_DYNAMIC_VISCOSITY_H_
+
+#include <string>
+#include <vector>
+#include <cmath>
+
+#include "MaterialLib/Fluid/FluidProperty.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/// A temperature dependent viscosity model.
+class VogelsLiquidDynamicViscosity : public FluidProperty
+{
+public:
+    /*!
+         \brief Viscosity defined by \f$e^{A+\dfrac{B}{C+T}}\f$
+         \param mat_id  0: Water
+                        1: Carbon dioxide
+                        2: Methane
+     */
+    VogelsLiquidDynamicViscosity(const unsigned mat_id)
+        : _abc(_constants[mat_id])
+    {
+    }
+
+    /// Get model name.
+    virtual std::string getName() const final
+    {
+        return "Liquid viscosity by Vogel's equation";
+    }
+
+    /// Get viscosity value.
+    /// \param var_vals Variable values in an array. The order of its elements
+    ///                 is given in enum class PropertyVariable.
+    virtual double getValue(const double var_vals[]) const final
+    {
+        return 1.e-3 *
```

Comment:
`1e-3` looks like unit transformation. Is it really needed in a pure SI setting?